src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -598,6 +598,16 @@ public abstract class AbstractCompilerMojo extends AbstractMojo {
 
     @Parameter(defaultValue = "false", property = "maven.compiler.showCompilationChanges")
     private boolean showCompilationChanges = false;
+
+    /**
+     * Timestamp for reproducible output archive entries, either formatted as ISO 8601
+     * <code>yyyy-MM-dd'T'HH:mm:ssXXX</code> or as an int representing seconds since the epoch (like
+     * <a href="https://reproducible-builds.org/docs/source-date-epoch/">SOURCE_DATE_EPOCH</a>).
+     * @since 3.11.1
```

Comment:
I think the next version will be `3.12.0`, at least based on the Jira Fix Version/s:

https://issues.apache.org/jira/browse/MCOMPILER-542?jql=project%20%3D%20MCOMPILER%20AND%20fixVersion%20%3D%203.12.0

Reply:
stupid me, thanks :)